### PR TITLE
instructor help page: select the radio buttons in the rubric question sample #2899

### LINKED ISSUE #2899: instructor help page: select the radio buttons in the rubric question sample
The radio button can be shown as selected.
image

### PR DESCRIPTION
**Pull request by manvijain**
Fixes #2899

### CODE REVIEW

File: .settings/com.google.appengine.eclipse.core.prefs [hunk 1]
```diff
@@ -3,6 +3,7 @@ filesCopiedToWebInfLib=appengine-api-1.0-sdk-1.9.4.jar|appengine-api-labs.jar|ap
 gaeDatanucleusVersion=v1
 gaeDeployDialogSettings=
 gaeIsEclipseDefaultInstPath=true
+gaeLaunchAppInBrowser=true
```
Comment (LowWeiLin) [inline .settings/com.google.appengine.eclipse.core.prefs]: This change seems irrelevant to the issue. Could you revert it?

Comment (manvijain) [inline .settings/com.google.appengine.eclipse.core.prefs]: Okay, I will do that.

**Comment by manvijain** [comment]
I undid the changes and made a new commit including only the relevant changes. Is it okay now?

**Comment by LowWeiLin** [comment]
The change in .settings/com.google.appengine.eclipse.core.prefs still seems to be there. Click on the files changed tab in this pull request to see what has been changed.

**Comment by manvijain** [comment]
Okay I thought reverting it removed the change. I will try again.

**Comment by manvijain** [comment]
I reverted that particular file to a previous commit. Is it okay now?

**Comment by LowWeiLin** [comment]
Yes, looks good now.
@damithc

**Comment by manvijain** [comment]
finally :D
